bazel/repositories.bzl

```diff
@@ -126,10 +126,10 @@ def proxy_wasm_cpp_host_repositories():
     maybe(
         git_repository,
         name = "v8",
-        # 10.7.193.13
-        commit = "6c8b357a84847a479cd329478522feefc1c3195a",
+        # 11.2.34
+        commit = "fb6dd9c9d5034ca221e895b01072b0269f2611e7",
```

Comment:
You need to update `//base/trace_event/common` below to a version that matches V8 as well.

See https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh (or just run it) to see how it's extracted.